tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py34-django110
```

Comment:
This project uses Django 1.8 - it might be useful to start looking at upgrading, but for the moment lets target the version in the requirements